src/Player.hpp

```diff
@@ -0,0 +1,34 @@
+#include <string>
+#include <vector>
+#include <memory>
+#include "Property.hpp"
+class Player{
+
+    std::string color_;
+    int money_;
+    int location_;
+    std::vector<Property> properties_;
+
+public:
+    Player()= delete;
+    Player(const Player &) = delete;
+    Player(Player &&) = delete;
+    ~Player() = default;
+    Player & operator=(const Player &) = delete;
+    Player & operator=(Player &&) = delete;
+
+    Player(const std::string & color);
+
+    std::string getColor() const;
+    int amountOfMoney() const;
+    int getLocation() const;
+    bool ownsProperty() const;
+
+    void setLocation(const int& location);
+    void addProperty(Property& property);
+
+
```

Comment:
Too many blank lines. Usually one line is enough. Maybe in some places where you need to split a file logically or by functionality you can use 2 blank lines.